README.md

```diff
@@ -22,6 +22,12 @@ npm install assert
 
 Due to differences between browsers, some error properties such as `message` and `stack` will be inconsistent. However the assertion behaviour is as close as possible to Node.js and the same error `code` will always be used.
 
+## Usage with bundlers that don't automatically include polyfills for Node.js APIs
+
+Bundlers like `webpack 5` and `Vite.js` (and possibly others) don't automatically include polyfills for Node.js APIs. This module depends on other Node.js APIs, so it won't work with these bundlers without also including a polyfill for the `process` Node.js global. You can use [this library](https://github.com/defunctzombie/node-process) to polyfill the `process` global.
+
+Note that this is not a problem with this package per se, this package will work without any manual configuration with `browserify` and `webpack 4`. It's just that some bundlers decide to allow-list all Node.js API specific packages, hence the incompatibility. See https://github.com/browserify/commonjs-assert/issues/55 for some more context.
```

Comment:
```suggestion
Note that this is not a flaw in this package - this package will work without any manual configuration with `browserify` and `webpack 4` and other working bundlers. Unfortunately, some bundlers decided to require an explicit allow-list of all Node.js API specific packages instead of having them work out of the box, hence the incompatibility. See https://github.com/browserify/commonjs-assert/issues/55 for some more context.
```